sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.12.0b2 (Unreleased)
+## 1.12.0b2 (2022-10-11)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
```

Comment:
I would consider including an "Other Changes" note about the fact that the VS Code credential can again be added to the DefaultAzureCredential chain, but is now opt-in instead of opt-out.

Reply:
Enable vs code cred can do nothing but fail. I don't think we want to mention it. We will add it back when we have the long-term solution implemented.